sql/privilege.md

```diff
@@ -291,7 +339,7 @@ Data in the `user` and `db` tables is also sorted when loaded into memory.
 
 The use of `%` in `tables_priv` and `columns_priv` is similar, but column value in `Db`, `Table_name` and `Column_name` cannot contain `%`. The sorting is also similar when loaded.
 
-#### Time of effect
+### Time of effect
 
 When TiDB starts, some privilege-check tables are loaded into memory, and then the cached data is used to verify the privileges. The system will periodically synchronize the `grant` table from database to cache. Time of effect is determined by the synchronization cycle. Currently, the value is 5 minutes.
```

Comment:
This statement is now out of date. Using the privilege management statement interface will [synchronously run flush privileges](https://github.com/pingcap/tidb/pull/8886) (for mysql compatibility).

```suggestion
When TiDB starts, some privilege-check tables are loaded into memory, and then the cached data is used to verify the privileges. Executing privilege management statements such as `GRANT`, `REVOKE`, `CREATE USER`, `DROP USER` will take effect immediately.

Manually editing tables such as `mysql.user` with statements such as `INSERT`, `DELETE`, `UPDATE` will not take effect immediately. This behavior is compatible with MySQL, and privilege cache can be updated with the following statement:
```

PTAL @tiancaiamao